src/Elcodi/Admin/PluginBundle/AdminPluginBundle.php

```diff
@@ -30,6 +30,6 @@ class AdminPluginBundle extends Bundle
      */
     public function getContainerExtension()
     {
-        return null;
+        return;
```

Comment:
This one can be skipped with `--fixers=-empty_return`, because this is applied by default on `symfony` level.